### PR TITLE
Test suit for teams and tags page

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the test suits of teams page

#
### Type of change :

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/Modals/FormModal/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  findByText,
+  fireEvent,
+  render,
+} from '@testing-library/react';
+import React from 'react';
+import FormModal from '.';
+
+const mockCancel = jest.fn();
+const mockSave = jest.fn();
+const mockForm = jest.fn().mockReturnValue(<p data-testid="form">data</p>);
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/Modals/FormModal/index.test.tsx:29]: Why do we need `data-testid` here?

File: catalog-rest-service/src/main/resources/ui/src/components/common/non-admin-action/NonAdminAction.test.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    expect(testButton).toBeInTheDocument();
+  });
+
+  it('Should restrict user if criteria does not match', async () => {
+    mockAuth.isAdminUser = false;
+    mockAuth.isAuthDisabled = false;
+
+    const { container } = render(
+      <NonAdminAction title="test popup" trigger="click">
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/common/non-admin-action/NonAdminAction.test.tsx:57]: I see the trigger is set to click but no click is fired using fireEvent, how is this test being checked?

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
Few comments, otherwise LGTM

File: catalog-rest-service/src/main/resources/ui/src/components/Modals/FormModal/index.test.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/components/Modals/FormModal/index.test.tsx:29]: for checking mocked component is rendered or not.

File: catalog-rest-service/src/main/resources/ui/src/components/common/non-admin-action/NonAdminAction.test.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/components/common/non-admin-action/NonAdminAction.test.tsx:57]: trigger value will be pass to popover component, and the tests of that component are cover under its test.

**Review by darth-coder00 — APPROVED** [review]

**Pull request merged**